### ISSUE TITLE
mdm --help undefined options

### ISSUE THREAD

**Issue by amitrahav** [chriswessels/meteor-deployment-manager#8]
output of mdm --help is:
```
info: Meteor Deployment Manager v0.1.3 loading at Thu Jul 02 2015 17:37:24 GMT+0300 (IDT)...
info: ================================
info: Usage: mdm [command] [options]
info: Commands:
info: undefined
info: Deploy the specified project to the remote server defined in the deployment configuration file.
info: undefined
info: Generate a sample deploy.json file in the current working directory.
info: undefined
info: Roll back the current deployment to the previously deployed build.
info: undefined
info: Start your application using Upstart.
info: undefined
info: Stop your application using Upstart.
info: undefined
info: Restart your application using Upstart.
info: ================================
info: Options:
info: undefined
info: Use this argument to print out this help text.
info: undefined
info: Use this argument for verbose logging to the console.
info: undefined
info: The path to your project. If not specified, the current working directory will be assumed.
info: undefined
info: The file name of your deployment configuration file. Defaults to deploy.json.
info: undefined
info: The environment that should be deployed to. Defaults to staging.
info: ================================

```

**Comment by robertpitt** [comment]
Same Here.

**Comment by jf1603** [comment]
+1

**Comment by chriswessels** [comment]
Thanks to @jf1603 for the fix in PR #9!
